### ISSUE TITLE
[feat] Ability to create (multicast) groups using the gateway website.

### ISSUE THREAD

**Issue by ODINServ** [zwave-js/zwave-js-ui#70]
Since https://github.com/zwave-js/zwavejs2mqtt/pull/25 is about to be merged.

It would be nice that we can create multicast groups using the gateway website.

The idea is to rename groups to association (since that is the current behavior of the groups tab).
After that then we could see a multicast request to a selection of nodes as a group.

The website should make sure that the correct nodes are matched with the correct command classes, and once a group is created using the website a new topic is created on which we can set values like if it is a normal device.

Groups can than also be detected via the Hass discovery in openhab / hass.

By doing this all zwave gouping logic is in the zwave2mqtr backend and not in the home automation tool.

The zigbee2mqtt also does this but not sure if the zwave protocol behaves the same.

So in short
* Ability to group nodes using the website 
* Ability to send commands via a new topic to that (multicast) group just like normal devices.

**Comment by AlCalzone** [comment]
Fixed by https://github.com/zwave-js/zwave-js-ui/pull/4382
